site/src/atomics/shadow.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Shadow Atomics
+description: Shadow related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Shadow Atomics
+
+Shadow atomics are used to add a shadow effect around the frame of an element.
+
+## Usage
+
+Here are examples of shadow atomics, ranging from `none` for no shadow to `extra-heavy` shadow.
+
+```html
+<div class="display-block border padding-sm">
+	<div class="box-shadow-none padding-md margin-bottom-md font-size-lg border-radius">
```

Comment:
super nit: you can move `font-size-lg` to the parent div and remove it from the sample ones just to shorten the number of classes on each of them for readability.

Reply:
Updated